nionswift_plugin/usim/InstrumentDevice.py

```diff
@@ -295,14 +351,13 @@ def __init__(self, instrument_id: str):
         self.property_changed_event = Event.Event()
         self.__camera_frame_event = threading.Event()
         self.__samples = [SampleSimulator.RectangleFlakeSample(), SampleSimulator.AmorphousSample()]
-        self.__sample_index = 0
+        self.__sample_index = 1
```

Comment:
For existing tests, can we leave this as 0 (flake)?

Reply:
I updated the tests to change the sample back to 0 (it was only the test that checks for the EELS ZLP intensity that failed with the sample being amorphous). It is just more convenient to have it default to 1 when you start Swift, because the amorphous sample is so much more useful for most tests. I would prefer leaving it like this, but unlike the stage size, it is not strictly needed.